src/config.rs

```diff
@@ -254,13 +267,16 @@ mod tests {
     async fn test_data() {
         let cfg = |cfg: &mut ServiceConfig| {
             cfg.data(10usize);
+            cfg.app_data(10usize);
```

Comment:
can you make this a different number/type as a guard against it doing the same as `Data<T>`.